sema/commons/j2/j2_functions.py

```
@@ -46,43 +52,77 @@
     # make rigid int
     if not isinstance(content, int):
         asint = int(str(content))
-        assert str(content) == str(
-            asint
-        ), "int format does not round-trip [ %s <> %s ]" % (
-            str(content),
-            str(asint),
-        )
+        if str(content) != str(asint):
+            raise ValueError(
+                f"int format does not round-trip [ {str(content)} <> {str(asint)} ]"
```

Ruff (0.8.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)